src/rich-text/commands/index.ts

```diff
@@ -462,7 +495,7 @@ export const createMenu = (options: CommonViewOptions): Plugin =>
             {
                 key: "redo",
                 command: redo,
-                dom: makeMenuIcon("Refresh", "Redo", "redo-btn", [
+                dom: makeMenuIcon("Refresh", "Refresh", "redo-btn", [
```

Comment:
This one should be left as "Redo"

```suggestion
                dom: makeMenuIcon("Refresh", "Redo", "redo-btn", [
```

Reply:
Redo and Undo both have keybindings as well, see [here](https://github.com/StackExchange/Stacks-Editor/blob/main/src/rich-text/key-bindings.ts#L48-L50).

Reply:
Do we ever show Redo and Undo as buttons?

Reply:
Yup, we show them on small screens + mobile

![image](https://user-images.githubusercontent.com/14169651/170059476-9b87441c-b537-4c45-9fe8-b8a2b6f356a8.png)

Reply:
Ah of course!